src/client/app/helpers/strings.ts

```diff
@@ -112,6 +112,8 @@ export class Strings {
 
     static readonly deleteSnippetConfirm = 'Are you sure you want to delete this snippet?';
 
+    static readonly triggerActionLabel = 'Trigger Suggest';
```

Comment:
When you think of this getting localized across multiple languages and people needing to figure out what this string means:  should it be more like `editorTriggerSuggestLabel` or `editorTriggerSuggestMenuItemLabel`?

Other than that, looks good.  Please push one final commit, and then I will merge it in.